The Fedora /etc/nsswitch.conf is based largely on the upstream
version with minor downstream distribution modifications for
use with SSSD and systemd.

diff --git a/nss/nsswitch.conf b/nss/nsswitch.conf
index f55358811495c9f1..14c3cde8080a848c 100644
--- a/nss/nsswitch.conf
+++ b/nss/nsswitch.conf
@@ -1,7 +1,7 @@
 #
 # /etc/nsswitch.conf
 #
-# An example Name Service Switch config file. This file should be
+# Name Service Switch config file. This file should be
 # sorted with the most-used services at the beginning.
 #
 # Valid databases are: aliases, ethers, group, gshadow, hosts,
@@ -52,18 +52,20 @@
 # shadow:    db files
 # group:     db files
 
-# In alphabetical order. Re-order as required to optimize peformance.
+# In order of likelihood of use to accelerate lookup.
+passwd:     sss files
+shadow:     files sss
+group:      sss files
+hosts:      files dns myhostname
+services:   files sss
+netgroup:   sss
+automount:  files sss
+
 aliases:    files
 ethers:     files
-group:      files
 gshadow:    files
-hosts:      files dns
 initgroups: files
-netgroup:   files
 networks:   files dns
-passwd:     files
 protocols:  files
 publickey:  files
 rpc:        files
-shadow:     files
-services:   files
